Simplenote/NSMutableAttributedString+Styling.m

```diff
@@ -3,6 +3,10 @@
 #import "Simplenote-Swift.h"
 
 
+// Note: Capture Group Mark I: We're only interested in replacing the `- [ ]` marker, not the leading spaces
+//
```

Comment:
Just curious,  Why two slashes `//`  here but three slashed `///` here https://github.com/Automattic/simplenote-ios/pull/708/files#diff-5ff7a2f45a87242564a5548454b30c2bR9

Reply:
Should definitely document this somewhere! Overall goal is as follows (there are probably few inconsistencies here and there, though).

A. Top level declarations: double `//`
B. Method documentation: three `///`


Sample:

```
// MARK: - NSRegularExpression
//
extension NSRegularExpression {

    /// Matches Checklists at the beginning of each line
    ///
    @objc
    static let regexForChecklists: NSRegularExpression = {
//...
    }()

```

Thanks for stepping by @metrue!!

Reply:
@jleandroperez Thanks for the clear explanation here.